website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -25,6 +25,12 @@ In previous versions, Consul emitted redundant state store usage metrics that co
 
 As of Consul v1.19, Consul does not emit the redundant metric with the double `consul.consul` in its name. Any monitoring alerts and dashboards that you may have utilizing these metrics may require edits to update to the simplified metric name.
 
+### Legacy API Gateway Removal
```

Comment:
We should remove the no-longer-existing stanza from the Helm chart docs: https://developer.hashicorp.com/consul/docs/k8s/helm#v-apigateway

Reply:
Updated